src/ui/screens/SendFundScreen.tsx

```diff
@@ -65,7 +66,7 @@ export function SendFundScreen({navigation, route}: Props) {
               keyboardType="decimal-pad"
               value={amount}
               onFocus={() => setAmount('')}
-              onChangeText={(nextValue) => setAmount(nextValue.replace(/[^(\d+).(\d+)]/g, ''))}
+              onChangeText={(nextValue) => setAmount(decimalKeypad(nextValue))}
```

Comment:
I have tested this on my device and it does not work. Here we can just do `nextValue.replace(',', '.')`, the other regex is not needed since the keyboardType will not allow typing letters. We should also add `contextMenuHidden={true}` to disallow copy/paste any other string here

What we need to do in these forms is adding validation/error handling. If the amount or the address are not correct the transaction will throw an error

Reply:
@franjs  for now, contextMenuHidden={true} has been added to avoid copy-paste, and the decimalKeypad method is improved